app/models/account.rb

```diff
@@ -64,6 +64,14 @@ def self.lookup_by_user_account_uuid(user_uuid)
     Account.where(icn: user_account.icn).first
   end
 
+  def mpi_profile
+    resp = MPI::Service.new.find_profile_by_identifier(
```

Comment:
What would you estimate the magnitude of the number of jobs with this being? Calling MPI is a nice solution to not have to provide SSN across sidekiq params, but want to make sure we aren't DDOS'ing the service

Reply:
≈ 100/day